<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1797</width>
    <height>1009</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>medit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="gb_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="gb_left_splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="tv_files">
        <property name="font">
         <font>
          <family>Source Code Pro Semibold</family>
         </font>
        </property>
       </widget>
       <widget class="QListWidget" name="lst_recent"/>
       <widget class="QPlainTextEdit" name="txt_log"/>
      </widget>
      <widget class="MEditor" name="txt_editor"/>
      <widget class="MView" name="wv_rendered" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MView</class>
   <extends>QWidget</extends>
   <header>medit.mview</header>
  </customwidget>
  <customwidget>
   <class>MEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>medit.meditor</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
